pod-operation/src/utils/gpio.rs

```diff
@@ -0,0 +1,13 @@
+#[non_exhaustive]
```

Comment:
Good work pointing out that this list is currently non-exhaustive. Will we be removing this once all pins have been accounted for?

Also, is there any pro to using this versus just a comment?